client/checkout/woopay/express-button/__tests__/index.test.js

```diff
@@ -26,25 +25,35 @@ describe( 'renderWooPayExpressButton', () => {
 	expressButtonContainer.setAttribute( 'id', 'wcpay-woopay-button' );
 
 	beforeEach( () => {
-		getConfig.mockReturnValue( 'foo' );
+		document.body.innerHTML = '';
 	} );
 
-	test( 'render the button component in placeholder', () => {
+	test( 'render the button component in placeholder', async () => {
 		document.body.appendChild( expressButtonContainer );
 
 		// trigger window load.
-		window.dispatchEvent( new Event( 'load' ) );
+		await act( async () => {
+			window.dispatchEvent( new Event( 'load' ) );
+
+			// giving React some time to render
+			await new Promise( ( resolve ) => setTimeout( resolve, 0 ) );
+		} );
 
 		expect(
-			screen.queryByText( 'WooPay Express Button' )
+			await screen.findByText( 'WooPay Express Button' )
```

Comment:
When some text appears, `findBy*` is the appropriate query to use, because it retries